rpc/master/service_server.go

```diff
@@ -65,12 +70,18 @@ func (s *Server) GetService(serviceID string, svc *service.Service) error {
 }
 
 // GetEvaluatedService returns a service where an evaluation has been executed against all templated properties.
-func (s *Server) GetEvaluatedService(request EvaluateServiceRequest, svc *service.Service) error {
-	result, err := s.f.GetEvaluatedService(s.context(), request.ServiceID, request.InstanceID)
+func (s *Server) GetEvaluatedService(request EvaluateServiceRequest, response *EvaluateServiceResponse) error {
+	svc, err := s.f.GetEvaluatedService(s.context(), request.ServiceID, request.InstanceID)
+	if err != nil {
+		return err
+	}
+
+	tenantID, err := s.f.GetTenantID(s.context(), request.ServiceID)
```

Comment:
I feel like this should be called in the facade, instead of in addition to the facade